File: Backend-2nf-Clothing-Web/2nf-API/Responses/SaleResponse.cs
namespace _2nf_API.Responses
{
    public class SaleResponse
    {
        public int Id {  get; set; }
        public DateTime Date { get; set; }
        public ClientResponse Client { get; set; }
        public List<SaleDetailResponse> Details { get; set; }
        public long PaymentId { get; set; }
        public bool Canceled { get; set; }
        public bool RefundPending { get; set; }
    }
}

File: Frontend-2nf-Clothing-Web/src/app/models/sales/responses/sale-response.ts
import { Client } from "../../clients/client";
import { SaleDetailResponse } from "./sale-detail-response";

export interface SaleResponse{
    id: number;
    date: Date;
    client: Client;
    details: SaleDetailResponse[];
    // shipment: ShipmentResponse;
    paymentId: number;
    canceled: boolean;
    refoundPending: boolean;
}